### PR TITLE
Update pytorch_lightning imports in examples

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

Makes it so we can run the examples with the new imports :))
Follow up to #(the PR from Jirka who changes the package structure)

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
I made sure I had fun coding 🙃

cc @borda @carmocca @akihironitta @justusschock @awaelchli

### CODE REVIEW

**Comment by Borda** [comment]
let's do it :rabbit:

**Review by Borda — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

File: .azure/gpu-tests-fabric.yml [hunk 1]
```diff
@@ -99,7 +99,10 @@ jobs:
         python .actions/assistant.py copy_replace_imports --source_dir="./tests" \
           --source_import="lightning.fabric,lightning.pytorch" \
           --target_import="lightning_fabric,pytorch_lightning"
-      displayName: 'Adjust tests'
+        python .actions/assistant.py copy_replace_imports --source_dir="./examples" \
+          --source_import="lightning.fabric,lightning.pytorch" \
+          --target_import="lightning_fabric,pytorch_lightning"
```
[the remark this answers is not in the document]
Comment (carmocca) [reply .azure/gpu-tests-fabric.yml]: Because azure CI installs with `PACKAGE_NAME=fabric`, meaning `lightning_fabric`

**Review by akihironitta — APPROVED** [review]

File: .azure/gpu-tests-fabric.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (awaelchli) [reply .azure/gpu-tests-fabric.yml]: Then let's merge it like this but after that we should undo this conversion magic in the CI. Now that the package is combined, it's only a misdirection and no longer necessary. Do you agree? cc @Borda

[the remark this answers is not in the document]
Comment (carmocca) [reply .azure/gpu-tests-fabric.yml]: I would keep it as it is if we don't duplicate our CI. `pytorch_lightning` is still the most downloaded package, so we should test that it doesn't break in these environments.

[the remark this answers is not in the document]
Comment (Borda) [reply .azure/gpu-tests-fabric.yml]: I would suggest parametrizing for both... give me a moment...

**Review by hhsecond — APPROVED** [review]

**Pull request merged**